src/components/layout/Header/Header.tsx

```diff
@@ -1,15 +1,15 @@
 import Navbar from "../../views/Navbar/Navbar";
 import LogoSection from "../../views/LogoSection/LogoSection";
 import { useSelector } from "react-redux";
-import { getDevice } from "../../../redux/deviseRedux";
+import { getDevice } from "../../../redux/deviceRedux";
 
 const Header = () => {
-  //const device = useSelector(getDevice);
-  //const isDesktop = device.isDesktop;
+  const device = useSelector(getDevice);
+  const isDesktop = device.isDesktop;
 
   return (
     <>
-      <Navbar /* isDesktop={isDesktop} */ />
+      <Navbar isDesktop={isDesktop} />
```

Comment:
Ideally adaptivity changes should live in css land. I've mentioned this in the previous comment as well. It is fine here. I personally do not expect you to rewrite a significant portion of this application to adapt to those changes, but going forward, this might be a _big-ish_ deal.